specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -1303,7 +1303,7 @@
           "format": "int32",
           "description": "Value associated with the air quality or pollution category. These values range from 1 to 6. 1 implying good conditions, 6 implying hazardous conditions."
         },
-        "type": {
+        "air_quality_type": {
```

Comment:
Should use x-ms-client-name instead. And `airQualityType` (will become `air_quality_type` in Python)